test-infra/prowjobs/cleanerupper/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM golang:alpine
+
+RUN apk add --no-cache git
+
+WORKDIR /tmp
+COPY main.go main.go
+RUN go get -d ./...
+RUN CGO_ENABLED=0 go build -o /cleanerupper .
+RUN chmod +x /cleanerupper
+
+FROM scratch
+
+ENV GOOGLE_APPLICATION_CREDENTIALS /etc/compute-image-tools-test-service-account/creds.json
```

Comment:
Do you have a link to documentation that suggests copying credentials into your docker image?

Reply:
We don't copy credentials into the image (granted the location is a bit odd, but mounts can be anywhere and this matches all our other containers), we mount them at node runtime:
https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/test-infra/prow/config.yaml#L136

I would like to go through and clean up these containers and move the environment setting to the config as that would make this much clearer

Reply:
ahhh that makes sense. Thanks!